gatsby/static/css/matrix-org.webflow.css

```diff
@@ -46,6 +46,15 @@ h6 {
   font-weight: 500;
 }
 
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+    scroll-margin-top: 74px;
```

Comment:
The `.mxnavbar` class does not appear to have a fixed height.
When I resize the window, I see that the navbar is either `60px` or `64px` tall.
Is `74px` an arbitrary size that will hopefully always be larger than the navbar?

Reply:
It was mostly a size that looks fine on my monitor 😅

But it's easy to use some media breakpoints to fix that scaling issue

Reply:
As it is currently, I think it would be valuable to add a comment noting how the value was chosen.
e.g. `/* An arbitrary size that has a ~10px buffer above the expected size of the navbar */`
But I agree that `@media` can be a better solution.

Reply:
Added the `@media` and the comment to it now :)